packages/@okta/i18n/src/properties/login.properties

```diff
@@ -769,6 +769,8 @@ oie.verify.webauthn.uv.required.instructions = Biometric verification or a PIN i
 oie.security.question.enroll.title = Set up security question
 oie.security.question.challenge.title = Verify with your Security Question
 
+## ODA/ Okta verify
+oie.app = Okta Verify
```

Comment:
No existing string we can use?

Reply:
Technically this is new for m1 since signed nonce uses it. Other names are 


factor.totpSoft.oktaVerify = Okta Verify
AND 
factor.oktaVerifyPush = Okta Verify

I guess for m2 we are going to pick the name for api anyway. U have any suggestions ? 

/cc @haishengwu-okta

Reply:
If this will change for m2, then i guess this is fine as is

Reply:
Leaning toward using `oie.*` for this case given `factor.*` doesn't make sense for OIE.
Also why `oie.app` instead of `oie.okta.verify` ?

Reply:
Checked with Wils and he was fine with this since it is a net new flow. OV (fastpass + totp + push)